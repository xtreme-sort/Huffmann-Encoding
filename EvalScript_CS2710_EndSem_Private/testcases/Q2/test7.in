DecodeMessage
(((((H,v),P),(C,a)),((K,l),((T,L),(s,N)))),((n,i),(W,(7,F))));
01011010010001000111110010011000011110010011101110011000110011001000000001110001010101100001101010111111101111111001110001101100101010101000110100000011011111011110010100000010110001000100010000100011011000101101001110111001000111101000011111111101100101011001110111010001111110010110101101000000001010111100111000001000000111100100011001001111000000111000001100011001101001101100111100000000101011111000010011000010100000101110100111101111011011000000011100110011110100000000001011001110000101100111001011111110010101110001101100011010010011000111000010101111101110011100011011010010101000100011010100011000110011001000111000100010111001101101111011001101010001010001010000100110000110011111000011000000010100011111011000101010010100001011000111011000000011001110001101101011100110000100111101101000101010000010001000011011010111000010000010101111111000000101011111001011010110011101111011010001011000010010001101100011101100010001110111011111110111101110111001101000100101100110010010000110000001100110000010000101011101001010101011110101101000101100100110111000001000000111000110110111110101001100100110110000011110011010100001011001101111111111000110000000110111101000111011001100001101010110000000011010000100000100100100111001100011011001100001000101110111110011100101011010011011001111100011110110010000001010111101111011000010100001000100000110001101111001101101011110100010001010010000100001000000011110000001010111010100111100101011100110101101011110111000100000101000111101101111101110000111100111110100001000001110000100001001001111001100000000010011010011100100011111100010001001000110001101011000000000110001010001110010100110101000010111110011100110101100111111111110011010110000010101100101101011000000111100110100110111001001001010111110110100000011011000000110000000001111001110101101011111000011011100000111110001001001100001001010010101010100010101010001101100010001001000000000100100001000100110000011111110001101111110100011010010011010011000000111100111100010011100110101100111001110111101010101011010111110011001101001100011110100001100111001000000110110101010110101110111110001100000011000101001010001010110101010111001010001000001100000100001101111000100110100000000010100101101011001110001000110111110101111010001101101111100001101011100111001101011001100110000010100000011010100001011011110010100001000101011010011111111100001001001010100000000111001010111101110011101100011010011110000110001100011000010011100010000001101111110010101111110010010000000000011111000010111111101101110011001111011000110110101100011110111100000000010010010111001101000001101100110111100000111000101101110000010001000010111000001000000111100110000101010111010101101011000000010101100011011111011000101010111000010011100000000000000110000000100010010111001010000000001011001110010001110010101100000100010110000101111110100100011100000110110001101011000111011111110000010000100110100011011110011110000101001011010000000010010011111101101000011011111111010010010000101011010111000110110100101110000000010110001001111000000001010100110000001000111001101010101100000010110111100110101010101010001100101111100001000100000111000101101110111100000111000111101111110100011101100000100110000001100100011010000001111101011010111100011111011100101110001000001010100111111101000100100100000010001011110100011110011101001001101100111111100100000000010000010101011010111101111111101010100010101110011110010000010000111100001010001111011000101101000101110100010101110010100000010001010011011100000001111111000100110100001011110000010001000110101001010101011010111111100110100001011100110000001111101101010000101000000101000000010000010110000011011100100010101101000101110111011100000110101110110001111110110110000100000110000010011111101101001110001010111100000011000011100011110100101010010100111100000011110100001100011100101011110100000101101011011001111101001101101101011000110000001001101101011001100011011110010001101000011011000000000000111101100010110001100001011010000001101111101010011011111101110000010011011111110110000010110101111011000101001000111000110101000001001001011100001001000101110110101100011010000101101100000011110001000001011000100011000000100001011000011011010001
Quit
