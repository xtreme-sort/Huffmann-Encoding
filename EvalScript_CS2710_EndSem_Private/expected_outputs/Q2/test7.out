liCCa7KWP7K77TTWKHsCiTalN7F7saTilKLHLFNCnvTKKKCaTliaiWKNKaF7WlT77nNWliLHPl7svHNCaCNHsvnTWnWWNHPlFvavKPsnFNLnH7TFKHPT7PT7lFWlsaTLCTsPlFssaLCiPPiKTTWKsCC7Li7WLKlPKCTaa7PnHiPFTlKivTsWHT7aLsTCNLPlHnKaLsPHiN7HiNniiT7FLPTCKLnsWCaiWF7Fs7LPCWTnnaHTWvvlWnilNliPTnW7vHsaLFlaCLnP7LKCWLFFnTHWFKsWTalTHLvHnnn7TLnWPPsFn7lLaTFnNTnvl7FTCnCCvnL7LiNKKlCPvHaWHisiaWlsLLNsCvKNLFsP7NivH7PvCNaHvaK7KNWCCKTLTHaPKslalvNn7LTFF7LTPlniiTv7LasKniNiiHLnvnHaWsiiNnasvFPCTClCilPlKLnKKnHCKCCTPFWaNWnLCLaHNaWCsLT7sFllLNnWLaP7naannviilLsFnTvnlCnlLlslPHWvvi7CLHvKiiT7CaNiNKLiFvissLTWTPKviKCWFCnCCiiaF7PClKHslNssWaKFvnTTCsCHWFWlNWKnHvFvN7W7TFTLiTNNHvClWLHWWL7vWCW7vPvsvHNavlsiLTHiTLFTllWPanHHaHCClWlHvT7KslTPPTCFWnnsvinLTsF7vvaKL7NvKiiHPCNWWnaNFKnnCiisaLC7HCWCNHPlaHKsLlTvL7LllKTiFvPH7CW7FH7aWFWnsWvaHTnLHNiLNPFslWCvlaFiPCKvPNKNaiCLnF7KHCvlLNNFlKlsNCvv7PKNTliPsnlslHKlasHN7CLvNHnKLKilLN7LvsTvFLKCnvKHnPTPiWKlLPs77visWaFLnCvnPaFLanlNHTanNKiKiaWHFKaPWlNKPLLnFiaLiTTvaLTWaNCaKaTHHFTlnTCWnviFlaNW7vaN7WvLNTlCanLKPClWPCC7WinLvLnv7CvTKTvvTaLP
